qtconsole/qtconsoleapp.py

```diff
@@ -10,6 +10,7 @@
 import os
 import signal
 import sys
+from packaging.version import parse
 from warnings import warn
 
```

Comment:
```suggestion
from warnings import warn

from packaging.version import parse
```

This is a third-party package, so it needs to go in a separate section.

Reply:
Sorry I swore it was a standard library, my bad!

Reply:
No problem. You would expect it to be, but (sadly) it isn't.